.stylelintrc

```diff
@@ -0,0 +1,178 @@
+{
+  "extends": "stylelint-config-standard",
+  "rules": {
+    "function-url-quotes": "always",
+    "string-quotes": "single",
+    "selector-attribute-quotes": "always",
+    "font-family-name-quotes": "always-where-recommended",
+    "at-rule-no-vendor-prefix": true,
+    "media-feature-name-no-vendor-prefix": true,
+    "property-no-vendor-prefix": true,
+    "selector-no-vendor-prefix": true,
+    "value-no-vendor-prefix": true,
+    "max-nesting-depth": 0,
+    "selector-max-compound-selectors": 3,
+    "selector-max-specificity": "0,4,0",
+    "at-rule-no-unknown": true,
+    "color-named": "never",
+    "declaration-no-important": true,
+    "selector-no-attribute": true,
+    "selector-no-id": true,
+    "selector-no-qualifying-type": true,
+    "selector-no-universal": true,
+    "font-weight-notation": "named-where-possible",
+    "no-browser-hacks": true,
+    "property-no-unknown": [ true, {
+      "ignoreProperties": ["composes"]
+    }],
+    "declaration-block-properties-order": [
+      "composes"
+    , "position"
```

Comment:
Ahhh seems like it might. We'll have to test this out. Mah bad on the formatting there.